src/app/certificate/certificate.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { User } from '../user/user.model';
+
+@Component({
+  selector: 'olx-certificate',
+  templateUrl: './certificate.component.html',
+  styleUrls: ['./certificate.component.css']
+})
+export class CertificateComponent implements OnInit {
+
+  @Input()
+  user: User;
+
+  @Input()
+  userPoints;
+
+  @Input()
+  maxPoints;
+
+  date = new Date().toDateString();
+
+  visibility = 'hidden';
+
+  constructor() {
+  }
+
+  ngOnInit() {
+  }
+
+  printToCart(printSectionId: string) {
```

Comment:
This seems to be a direct copy-paste of a Stackoverflow, including the typo (assuming it was meant to be "print card") that came from the plunker the question was based on 🙂

Please give the function a meaningful name. Also why width 600 and height 700?